src/qt/intro.cpp

```diff
@@ -302,7 +302,10 @@ void Intro::startThread()
 
     connect(executor, &FreespaceChecker::reply, this, &Intro::setStatus);
     connect(this, &Intro::requestCheck, executor, &FreespaceChecker::check);
-    /*  make sure executor object is deleted in its own thread */
```

Comment:
In commit "gui: Fix intro dialog when prune is toogled" (4824eeb91e8ee7ed2e46519f56d860635e7c4818)

Suggest not deleting this comment line assuming it is still applicable.

Reply:
This line
```
connect(thread, &QThread::finished, executor, &QObject::deleteLater);
```
seems to be a standard pattern in Qt framework. Yes, the comment is still applicable, but it does not provide any additional knowledge, IMHO.